lib/BaseLogger.js

```diff
@@ -56,6 +56,12 @@ class BaseLogger {
   }
 
   _writeLog(level, message) {
+    // We may have sent an Error;
+    // if we have, the stack is most useful.
+    if (message && message.stack) {
+      return this._writeLog(level, message.stack);
```

Comment:
I think it'd be helpful to also have the error message in the log - it might contain the application id etc.